### PR TITLE
Fix/multiple fixes3

### PR DESCRIPTION
**Pull request by szymonlesisz**
- format XRP reserve and ERC20 tokens amounts
- add key to tx list item
- add notifications to device settings
- remove "aaa" from sign/verify
- add close button to wallet settings
- disable native menu in electron app
- extend #630 : fix "device counter"

### CODE REVIEW

File: packages/suite/src/views/wallet/settings/Container.ts [hunk 1]
```diff
@@ -1,7 +1,7 @@
 import { connect } from 'react-redux';
 import { bindActionCreators } from 'redux';
 import { injectIntl } from 'react-intl';
-
+import { goto } from '@suite-actions/routerActions';
```
Comment (mroz22) [inline packages/suite/src/views/wallet/settings/Container.ts]: thought it is better to import is * as router actions :]

**Review by mroz22 — APPROVED** [review]
looks good!

File: packages/suite/src/views/wallet/settings/Container.ts (hunk 1 above)
Comment (szymonlesisz) [reply packages/suite/src/views/wallet/settings/Container.ts]: good catch

**Comment by mroz22** [comment]
nice, merging after ci is done

**Pull request merged**